ui/packages/atlasmap/src/Atlasmap/Atlasmap.tsx

```diff
@@ -164,19 +160,22 @@ export const Atlasmap: FunctionComponent<IAtlasmapProps> = ({
         handlers.onEditConstant({ value, valueType });
       },
       onDeleteConstant: handlers.onDeleteConstant,
-      onCreateProperty: handlers.onCreateProperty,
-      onEditProperty: (property) => {
+      onCreateSourceProperty: () => {
+        handlers.onCreateProperty(true);
+      },
+      onEditProperty: (property, isSource) => {
         const [leftPart] = property.split(" ");
-        const value = getPropertyValue(leftPart);
-        const valueType = getPropertyType(leftPart);
-        const scope = getPropertyScope(leftPart);
+        const valueType = getPropertyType(leftPart, isSource);
+        const scope = getPropertyScope(leftPart, isSource);
 
-        handlers.onEditProperty({
-          name: leftPart,
-          value,
-          valueType,
-          scope,
-        });
+        handlers.onEditProperty(
+          {
+            name: leftPart,
+            valueType,
+            scope,
+          },
+          true,
```

Comment:
`isSource` is needed by the `PropertiesTree` tree walker to get the lines correct so went with that and have a single createProperty() instead of createSourceProperty/ createTargetProperty.